cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
You could consider using `serializers.ModelSerializer` and try to rely on the implemented logic of the create and update methods.